paket.dependencies

```diff
@@ -10,6 +10,7 @@ nuget FSharpLint.Core prerelease
 nuget VSSDK.IDE.Deploy.14
 nuget FSharp.Management
 nuget FsPickler
+nuget FSharp.Interop.Dynamic
```

Comment:
It has been used extensively in this module https://github.com/fsprojects/VisualFSharpPowerTools/blob/e3b2531d83e008360c1fd6bcdb0bcfb25c9aaa31/src/FSharpVSPowerTools.Logic/FolderMenuCommands.fs.